app/web/__mocks__/nextFontMock.js

```diff
@@ -0,0 +1,12 @@
+module.exports = new Proxy(
+  {},
+  {
+    get: function getter() {
+      return () => ({
+        className: "className",
+        variable: "variable",
+        style: { fontFamily: "fontFamily" },
+      });
+    },
+  }
+);
```

Comment:
Next v13 has some new font features in next/font so just adding this now.

Reply:
Out of interest, are there any reason you're using a Proxy over an object with an getter on it?